Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -65,19 +65,14 @@ namespace AudioCommon
 		if (g_sound_stream)
 		{
 			UpdateSoundStream();
-			if (!g_sound_stream->Start())
+			if (g_sound_stream->Start())
 			{
-				ERROR_LOG(AUDIO, "Could not start backend %s, using %s instead",
-					  backend.c_str(), BACKEND_NULLSOUND);
-				delete g_sound_stream;
-				g_sound_stream = new NullSound();
```

Comment:
This will drop the "feature" to fallback to nullaudio if the configured audio backend fails. Not having an audio backend did break something in core iirc.